src/pybamm/config.py

```diff
@@ -21,20 +29,18 @@ def is_running_tests():  # pragma: no cover
     ):
         return True
 
-    # Check for GitHub Actions environment variable
-    if "GITHUB_ACTIONS" in os.environ:
-        return True
-
     # Check for other common CI environment variables
-    ci_env_vars = ["CI", "TRAVIS", "CIRCLECI", "JENKINS_URL", "GITLAB_CI"]
+    ci_env_vars = [
+        "GITHUB_ACTIONS",
+        "CI",
+        "TRAVIS",
+        "CIRCLECI",
+        "JENKINS_URL",
+        "GITLAB_CI",
+    ]
     if any(var in os.environ for var in ci_env_vars):
         return True
 
-    # Check for common test runner names in command-line arguments
-    test_runners = ["pytest", "unittest", "nose", "trial", "nox", "tox"]
-    if any(runner in arg.lower() for arg in sys.argv for runner in test_runners):
```

Comment:
This did not seem to work on Mac or Windows. I was seeing nox and pytest commands go right past this. I tested it by moving it to the top of the file and putting an assert below it.